MovieJournalMVC/Model/MovieReview.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  MovieReview.swift
+//  MovieJournalMVC
+//
+//  Created by Lauren Small on 3/12/19.
+//  Copyright © 2019 Lauren Small. All rights reserved.
+//
+
+import Foundation
+
+class MovieReview {
```

Comment:
For something simple like this I think a `struct` would work best. Additionally, you shouldn't create values with defaults like this. Instead, creating the value should use an initializer to set the values and then you can decide whether the value should be mutable later.

Reply:
I removed the defaults (empty strings) and changed the class to a struct and modified the update() function in the Store class as a result because of this error:

![image](https://user-images.githubusercontent.com/42355058/54958371-dce9ab00-4f2b-11e9-92e8-77effd24246c.png)

Reply:
The update function now looks like this:

func update(entry: MovieReview, with title: String, review: String) {
        
        var entry = entry
        
        entry.title = title
        entry.review = review
    }